app/controllers/users/folders/binaries/comment_likes_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Users::Folders::Binaries::CommentLikesController < ApplicationController
+  def create
+    # binding.pry
+    # if params[:binary]
+    binary = Binary.find_by(name: params[:binary_name])
+    user = binary.folder.owner
+    comment = binary.comments.first
+    like = comment.likes.new(user: user)
+    like.update(like: true)
```

Comment:
@jk1dd you don't need to do this in a separate line - you could put it in the arguments on line 8 above 👍